internal/pipeline/fn.go

```diff
@@ -45,10 +45,7 @@ type KRMFn interface {
 // Built-in functions that are compiled into kpt
 
 // annotator is built kpt function for annotating KRM resources.
-type annotator struct {
-	key   string
-	value string
-}
+type annotator struct{}
```

Comment:
I made this comment somewhere else: Without having TODO(s), it's not clearly why there's an example function in a file called fn.go, and it's hard for reviewers to actually give feedback. If we need a placeholder, then it needs to be communicated properly.